packages/react-components/react-migration-v0-v9/stories/List/Selectable.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import { Image } from '@fluentui/react-components';
+import { List, ListItem, useListSelection } from '@fluentui/react-migration-v0-v9';
+
+import * as React from 'react';
+
+type Item = {
+  key: string;
+  media: string;
+  header: string;
+  headerMedia: string;
+  content: string;
+};
+
+const items: Item[] = [
+  {
+    key: 'robert',
+    media: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/RobertTolbert.jpg',
+    header: 'Robert Tolbert',
+    headerMedia: '7:26:56 AM',
+    content: 'Program the sensor to the SAS alarm through the haptic SQL card!',
+  },
+  {
+    key: 'celeste',
+    media: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/CelesteBurton.jpg',
+    header: 'Celeste Burton',
+    headerMedia: '11:30:17 PM',
+    content: 'Use the online FTP application to input the multi-byte application!',
+  },
+  {
+    key: 'cecil',
+    media: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/CecilFolk.jpg',
+    header: 'Cecil Folk',
+    headerMedia: '5:22:40 PM',
+    content: 'The GB pixel is down, navigate the virtual interface!',
+  },
+];
+
+export const Selectable = () => {
+  const selection = useListSelection({ selectionMode: 'single' });
```

Comment:
would it be possible to change this example to show how they can control the selection? In reality, all states in an application need to be controlled, so that they can plug it into necessary layers.. auto controlled is only useful for demo purposes.